superset-frontend/src/components/DvtCheckbox/dvt-checkbox.module.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { styled } from '@superset-ui/core';
+
+interface StyledCheckboxProps {
+  checked: boolean;
+  disabled: boolean;
+}
+
+const StyledCheckbox = styled.div`
+  display: flex;
+  gap: 9px;
+  align-items: center;
+`;
+
+const StyledCheckboxInput = styled.div<StyledCheckboxProps>`
+  position: relative;
+  width: 24px;
+  height: 24px;
+  border: 2px solid ${({ theme }) => theme.colors.dvt.primary.base};
+  border-radius: 7px;
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
```

Comment:
"cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};"
bu "StyledCheckbox" yerine taşınır mısın. checkbox ve label beraber "pointer" gözüksün